src/index.html

```diff
@@ -17,6 +29,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
```

Comment:
Consider adding a `<h1>` tag within the header to improve accessibility and SEO. This will help screen readers and search engines understand the main topic of the page.